### PR TITLE
Feature/program version

### PR DESCRIPTION
**Pull request by jemten**
### This PR fixes:

- removed some references to condas base environment
- added qcc_regex for sambamba, picard and gatk

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Sv_reformat.pm [hunk 1]
```diff
@@ -225,7 +226,7 @@ sub analysis_reformat_sv {
     my $FILEHANDLE = IO::Handle->new();
 
     ## Creates recipe directories (info & data & script), recipe script filenames and writes sbatch header
-    my ( $recipe_file_path, $recipe_info_path ) = setup_script(
+    my ( $recipe_file_path, $recipe_info_path, $test ) = setup_script(
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Sv_reformat.pm]: How is the $test variable used here?

File: lib/MIP/Recipes/Analysis/Sv_reformat.pm [hunk 2]
```diff
@@ -380,6 +381,14 @@ sub analysis_reformat_sv {
 
     if ( $recipe_mode == 1 ) {
 
+        # Save STDERR in Sample info
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Sv_reformat.pm]:
```suggestion
        # Save STDERR in sample info
```

File: t/check_executable_in_path.t [hunk 3]
```diff
@@ -143,80 +107,11 @@ trap {
...
-    )
+    );
 };
 
-## Then FATAL message should broadcast
-like( $trap->stderr, qr/FATAL/xms,
-    q{No bin and executable in load env cmd - Throw FATAL log message} );
-
+## Then exit and trow FATAL message
+ok( $trap->exit, q{exit on no bin} );
```
Comment (henrikstranneheim) [inline t/check_executable_in_path.t]:
```suggestion
ok( $trap->exit, q{Exit on no bin} );
```

File: t/check_executable_in_path.t [hunk 4]
```diff
@@ -143,80 +107,11 @@ trap {
...
-        }
-    )
+    );
 };
 
-## Then FATAL message should broadcast
-like( $trap->stderr, qr/FATAL/xms,
-    q{No bin and executable in load env cmd - Throw FATAL log message} );
-
+## Then exit and trow FATAL message
```
Comment (henrikstranneheim) [inline t/check_executable_in_path.t]:
```suggestion
## Then exit and throw FATAL message
```

File: lib/MIP/Recipes/Analysis/Sv_reformat.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Recipes/Analysis/Sv_reformat.pm]: Ah! That shouldn't be there any more

Comment (jemten) [inline lib/MIP/Recipes/Analysis/Sv_reformat.pm]:
```suggestion
    my ( $recipe_file_path, $recipe_info_path ) = setup_script(
```

**Pull request merged**